scripts/st2bootstrap-deb.sh

```diff
@@ -1,528 +1,1046 @@
+
+        --stable|-s:    Install StackStorm packages from the stable repository. (default)
+                        Packages are officially supported and production ready.
+                        The stable option is mutually exclusive with the unstable option.
+
+        --unstable|-u:  Install StackStorm packages from the unstable repository.
+                        Daily or Promoted packages built after passing end-to-end testing from the StackStorm development branch.
+
+        --staging:      Install StackStorm packages from the staging-<stable|unstable> repository.
+                        This option is combined with the stable/unstable option.
+                        staging-stable packages are release candidate made available for testing during the StackStorm release process.
+                        staging-unstable experimental packages that are built from the latest development branch that have passed unit testing.
+
+        --dev=*:        Install StackStorm from Continuous Integration artifact.
+                        The pamameter takes the git repository name and build number - <repo_name>/<build_num>.  E.g. --dev=st2/5646
+                        Do not use this option unless you understand what you're doing.
+
+        --no-mongodb    Disable the installation procedure for MongoDB on the system.
+
+        --no-rabbitmq   Disable the installation procedure for RabbitMQ on the system.
+
+        --no-redis      Disable the installation procedure for Redis on the system.
+
+        --no-st2chatops Disable the installation procedure for st2 chatops on the system.
+
+        --no-st2web     Disable the installation procedure for st2 web ui on the system.
+
+EOF
+}
+function centre()
+{
+    LINE_LEN="$1"
+    TEXT="$2"
+    OUTPUT=""
+    
+    if [[ ${#TEXT} -lt ${LINE_LEN} ]]; then
+        LS=$(( (LINE_LEN - ${#TEXT}) / 2 ))
+        OUTPUT+=$(printf "%0.s " $(seq 0 $LS))
+        OUTPUT+="$TEXT"
+        RS=$(( LINE_LEN - ${#OUTPUT} ))
+        OUTPUT+=$(printf "%0.s " $(seq 0 $RS))
+    fi
+    
+    echo "${OUTPUT}"
+}
+# colour echo (ref https://stackoverflow.com/questions/4842424/list-of-ansi-color-escape-sequences)
+function cecho()
+{
+    if [[ "$1" == "-n" ]]; then
+        # No carrage return
+        local NCR="$1"; shift
+    else
+        local NCR=""
+    fi
+    local C="$1";
+    local MSG="$2"
+    echo $NCR -e "${C}${MSG}\e[0m"
+}
+function heading()
+{
+    local COLS=$(stty size | cut -d' ' -f2)
+    if [[ -n "$COLS" ]]; then
+        HEADING=$(centre $((COLS - 1)) "$1")
+    else
+        HEADING="$1"
+    fi
+    echo
+    cecho "\e[38;5;208m\e[48;5;238m\e[1m" "$HEADING"
+    echo
+}
+function echo.info()
+{
+    cecho "\e[37;1m" "$1"
+}
+function echo.warning()
+{
+    cecho "\e[33;1m" "$1"
+}
+function echo.error()
+{
+    cecho "\e[31;1m" "$1" >/dev/stderr
+}
+###############[ SCRIPT PARAMETER SETUP ]###############
+setup_install_parameters()
+{
+    
```

Comment:
```suggestion

```
Dang it. The Jinja comments are leaving whitespace. I guess wherever we just added `{#` it should probably be `{#-`. 😔

Reply:
Added left trim of white spaces like you suggested and it's cleaned up the excess white space.